autoprecompiles/src/optimizer.rs

```diff
@@ -95,46 +88,14 @@ fn optimization_loop_iteration<T: FieldElement>(
     Ok(system)
 }
 
-pub fn optimize_pc_lookup<T: FieldElement>(
-    mut machine: SymbolicMachine<T>,
-    opcode: u32,
-    pc_lookup_bus_id: u64,
-) -> SymbolicMachine<T> {
-    let mut first_pc = None;
-    machine.bus_interactions.retain(|bus_int| {
```

Comment:
Now, all PC lookups should be removed by existing optimizations, because all values on the LHS should be known at compile time and the PC lookup is stateless.

Reply:
done (at the end of `optimize`).